sql/src/test/java/io/crate/execution/jobs/CountTaskTest.java

```diff
@@ -78,7 +78,7 @@ public void testClose() throws Exception {
 
         // on error
         future = new CompletableFuture<>();
-        when(countOperation.count(eq(txnCtx), anyMap(), any(Symbol.class))).thenReturn(future);
+        when(countOperation.count(eq(txnCtx), any(), any(Symbol.class))).thenReturn(future);
```

Comment:
just curious: why isn't `anyMap` working here? (also I think there is now an unused import ;)